com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineMaterialUpgrader.cs

```diff
@@ -206,6 +206,7 @@ bool ShouldUpgradeShader(Material material, HashSet<string> shaderNamesToIgnore)
             return !shaderNamesToIgnore.Contains(material.shader.name);
         }
 
+        /// <inheritdoc/>
```

Comment:
Not sure why this is needed as well, even though this is public here the class is internal. Maybe another shortcoming in the tooling?

Reply:
It actually wasn't needed. I simply added it in case we make it public.